.reek

```diff
@@ -51,8 +51,10 @@ NilCheck:
 LongParameterList:
   exclude:
     - IdentityLinker#optional_attributes
-    - VendorValidatorJob#perform
     - Idv::VendorResult#initialize
+    - Idv::ProfileJob#perform
```

Comment:
I did not check the reek box because I disabled reek here.

The easiest way to fix this in the code is to use a `params` hash instead of keyword arguments. I'm not a fan of that since it is much harder to know what args the method actually takes.

The other thing I thought of was to create a struct (something like `Idv::JobArguments`) and put all of the args on that, but that seems like overkill and unnecessarily complicated so I just disabled the reek check.